### DISCUSSION TITLE
Adding Domino Block to OpenSpiel

### DISCUSSION THREAD

**Discussion by morLev** [deepmind/open_spiel#991] [Ideas]
Hello,
I would like to implement a new game to your repo: Domino Block implemented in python: https://en.wikipedia.org/wiki/List_of_domino_games#Block_Game
1. Will you accept this contribution?
2. Should I create a game state that automatically plays in situations that have one possible action to take? or in other words, does my game tree can have player nodes with only one brunch? (occurs a lot in Domino-block)

**Comment by lanctot** [comment]
Cool!

1. Absolutely! :) We will review it and may require some modifications before we do, but usually these are not too much to do.
2. Entirely up to you. We have games in the library that went one way, and others that went the other. All the algorithms will still work even if you have legal move lists of size 1. If it's your first game implementation, my advice is to do whatever is easiest for you.

**Reply by morLev** [reply]
Hi,
PR at https://github.com/deepmind/open_spiel/pull/1018

**Reply by lanctot** [reply]
Great! There have been some issues with the tests lately, so you will have to pull changes from master once #1017 is merged. I will ping you on the PR thread once it's ready, so that the tests can be triggered before we take a look.
